types/react-native-dotenv.d.ts

```diff
@@ -11,7 +6,7 @@ declare module '@env' {
   export const MAPBOX_API_TOKEN: string;
   export const MAPBOX_STOP_PLACES_STYLE_URL: string;
   export const AUTHORITY: string;
-  export const APP_ORG: AppOrgs;
+  export const APP_ORG: string;
```

Comment:
Kan importere AppOrgs som en del av deklerasjonen her?

Reply:
Hvis du importer enum in i d.ts får man "Cannot find module '@env' or its corresponding type declarations.ts(2307)
" i andre filer som bruker env

Reply:
Men vil ikke kunne importeres dynamisk på et vis her?
```suggestion
  export const APP_ORG: import('./app-orgs').AppOrgs;
```

Spørsmålet er vel derimot om det er korrekt modellert uten at det trengs decoding/mapping.